x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks.tsx

```diff
@@ -126,10 +128,14 @@ const BenchmarkSearchField = ({
 };
 
 export const Benchmarks = () => {
+  const [pageSizes, setPageSize] = useLocalStorage<number>(
+    LOCAL_STORAGE_PAGINATION_BENCHMARK_KEY,
+    10
```

Comment:
I've noticed the other hooks are passing the page size from the URL query. was wondering why this is the only exception to that.
Also, I'll need to checkout the PR to make sure it doesn't effect the behavior between pages, sadly I won't have time to do that today.